receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers.go

```diff
@@ -306,3 +308,37 @@ func (ghs *githubScraper) getCommitInfo(
 func getAge(start time.Time, end time.Time) int64 {
 	return int64(end.Sub(start).Seconds())
 }
+
+func (ghs *githubScraper) getCVEs(
+	ctx context.Context,
+	client graphql.Client,
+	repo string,
+) (*getRepoCVEsResponse, error) {
+	cves, err := getRepoCVEs(ctx, client, ghs.cfg.GitHubOrg, repo)
+	if err != nil {
+		ghs.logger.Sugar().Errorf("error getting CVE data from repo: %v", zap.Error(err))
+	}
+
+	return cves, err
+}
+
+func mapSeverities(
```

Comment:
Could this function have an issue & error out with anything going on inside?

Reply:
I just did a check to ensure a bad value isn't being mapped. Added a test for this case as well.